### DISCUSSION TITLE
Possibility to perform a snapshot

### DISCUSSION THREAD

**Discussion by jodewee** [dougiteixeira/proxmoxve#278] [Feature request] [answered]
Nice feature would be the possibility to take a snapshot of a vm or container

**Comment by danielholm** [comment]
I agree. As well as running backup jobs in general

**Comment by michalpulda** [comment]
This would be great! <3

**Comment by EpicLPer** [comment]
Any updates on this? :(

**Comment by mZ738** [comment, marked as answer]
A `Create snapshot` button exists per VM and container since 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)): a disk-only snapshot named `homeassistant_<date>_<time>`, described as created by Home Assistant, needing `VM.Snapshot`. Off by default like the other buttons that change things; it appears only where the credentials hold the privilege. Backups came along with it — buttons, a `proxmoxve.backup` action and a scheduled-backup blueprint, see [Actions](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/actions.md). Closing as done.
